### PR TITLE
Added Big Android BBQ Amsterdam

### PR DESCRIPTION
**Pull request by welsinga**

### CODE REVIEW

File: _conferences/big-android-bbq-amsterdam-2015.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Big Android BBQ Amsterdam"
```
Comment (JakeWharton) [inline _conferences/big-android-bbq-amsterdam-2015.md]: You don't need the location in the name. We will always render the location with the name.

Comment (welsinga) [inline _conferences/big-android-bbq-amsterdam-2015.md]: Problem was that is already had a file with the same name

Comment (JakeWharton) [inline _conferences/big-android-bbq-amsterdam-2015.md]: The file name needs the location, just not the name here.

Comment (welsinga) [inline _conferences/big-android-bbq-amsterdam-2015.md]: New pull request?

Comment (welsinga) [inline _conferences/big-android-bbq-amsterdam-2015.md]: New pull request?

Comment (JakeWharton) [inline _conferences/big-android-bbq-amsterdam-2015.md]: If you add another commit with the change and push it to your repo it'l update this pull request.

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**